src/management/Akka.Management/Routes/ClusterClientReceptionistRouteProvider.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System.Threading.Tasks;
+using Akka.Actor;
+using Akka.Cluster;
+using Akka.Http.Dsl;
+using Akka.Management.Dsl;
+using Akka.Remote;
+using Ceen;
+using Newtonsoft.Json;
+using Route = System.ValueTuple<string, Akka.Http.Dsl.HttpModuleBase>;
+
+namespace Akka.Management.Routes;
+
+public class ClusterClientReceptionistRouteProvider: IManagementRouteProvider
+{
+    public Route[] Routes(ManagementRouteProviderSettings settings)
+    {
+        return new HttpClusterClientReceptionistRoutes().Routes;
+    }
+}
+
+internal class HttpClusterClientReceptionistRoutes : HttpModuleBase
+{
+    public Route[] Routes => new Route[] { ("/cluster-client/receptionist", this) };
+    
+    public override async Task<bool> HandleAsync(IAkkaHttpContext context)
+    {
+        if (context.HttpContext.Request.Method.ToLowerInvariant() != "get")
+            return false;
+        
+        // Check that remoting is in effect
+        if (((ExtendedActorSystem)context.ActorSystem).Provider is not IRemoteActorRefProvider actorProvider)
+        {
+            context.HttpContext.Response.StatusCode = HttpStatusCode.ServiceUnavailable;
+            var response = JsonConvert.SerializeObject(new
+            {
+                error = new
+                {
+                    reason = "not available",
+                    message = "Remoting is not available"
+                },
+                code = (int)HttpStatusCode.ServiceUnavailable,
+                message = "Remoting is not available"
+            });
+            await context.HttpContext.Response.WriteAllJsonAsync(response);
+            return true;
+        }
+        
+        // Check that clustering is in effect
+        if (((ExtendedActorSystem)context.ActorSystem).Provider is not IClusterActorRefProvider)
+        {
+            context.HttpContext.Response.StatusCode = HttpStatusCode.ServiceUnavailable;
+            var response = JsonConvert.SerializeObject(new
+            {
+                error = new
+                {
+                    reason = "not available",
+                    message = "Clustering is not available"
+                },
+                code = (int)HttpStatusCode.ServiceUnavailable,
+                message = "Clustering is not available"
+            });
+            await context.HttpContext.Response.WriteAllJsonAsync(response);
+            return true;
+        }
+        
+        // Check that ClusterClientReceptionist is available
+        var config = context.ActorSystem.Settings.Config.GetConfig("akka.cluster.client.receptionist");
+        if(config is null)
+        {
+            context.HttpContext.Response.StatusCode = HttpStatusCode.ServiceUnavailable;
```

Comment:
Return a 503 - Service Unavailable if `akka.cluster.client.receptionist` HOCON config is not available. 
We're not referencing Akka.Cluster.Tools NuGet, no type checking is nescessary, it is the ClusterClient job to check that this actor path is actually alive.

Reply:
If the default HOCON from the `ClusterClient` package gets loaded but there's no actual receptionist running, what happens then?

Reply:
Rephrasing my question: is loading the default HOCON from the ClusterClient module enough to flip this bit? Or does the user have to actively populate it? The correct answers must be 1. "no" 2. "yes"

Reply:
This is strictly `Akka.Management`, using `ClusterClient` does not start `Akka.Management`.

For the discovery feature to work, the user has to start `Akka.Management` on the `ClusterClientReceptionist` side.

```
builder.
  WithAkkaManagement() // <-- This MUST be used
  WithClusterClientReceptionist(); <-- This DOES NOT load Akka.Management
```